README.md

```diff
@@ -210,7 +210,7 @@ Shaka Player supports:
   - SubViewer (SBV)
     - UTF-8 encoding only
 
-Subtitles are rendered by the browser by default.  Applications can create a
+Subtitles are rendered by the browser by default. Applications can create a
```

Comment:
It's silly, but I was taught to type (in Texas, in the 90s) with two spaces after a period. I know now that was never the English standard in most countries, and that it's no longer the standard today in any major American style guide. But it's a very difficult habit to break. I'm typing single-spaces now in this message, and I'm having to concentrate much more than I'm used to. :smile:

So on this point, you are correct again. But this is not a meaningful change, in my opinion, since rendering markdown on the web will collapse those extra spaces anyway.

We would love it if you could find some other way to contribute.  If you're more interested in contributing to the documentation, I'm sure there are real improvements you could make.  I find that new team members, having a fresh perspective, can point out issues that the rest of us have overlooked due to our long familiarity with the project.  Try to find something more substantive to contribute next time.  Thanks!